Utilities/js.cpp

```diff
@@ -106,24 +106,44 @@ enum class PrintTarget {
     StandardOutput,
 };
 
-static ErrorOr<void> print(JS::Value value, PrintTarget target = PrintTarget::StandardOutput)
+static ErrorOr<NonnullOwnPtr<Stream>> safely_acquire_print_stream(PrintTarget target)
```

Comment:
I don't really like "safely acquire X" here, makes it sound like there's an explicit lock somewhere you're trying to acquire "safely".
What the function does feels more like a `flushed_print_stream(PrintTarget)` to me.